.github/workflows/publish-image.yaml

```diff
@@ -0,0 +1,29 @@
+name: Release Docker images
+on:
+  push:
+    branches:
+    - 'develop'
+
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
```

Comment:
@tjholm I don't have permission to view/set these. Can you please?

Reply:
@asalkeld they are set at the org level so are already available for this repo.